app/assets/config/manifest.js

```diff
@@ -1 +1,3 @@
 //= link_tree ../builds
+//= link administrate-field-jsonb/application.css
+//= link administrate-field-jsonb/application.js
```

Comment:
Because we moved the admin only settings to the admin area, previously we handled it by hiding the fields form non-admin in the `/settings` path. These are used by the adminstrate-field-jsonb gem to style and add js for the `onboarding_allowed` attribute of an organization, which is a jsonb type.